pages/common/catimg.md

```diff
@@ -0,0 +1,21 @@
+# catimg
+
+> Image printing in the terminal.
+> See also: `pixterm` and `chafa`.
+> More information: <https://github.com/posva/catimg>.
+
+- Print a JPEG, PNG, or GIF to the terminal:
+
+`catimg {{path/to/file}}`
+
+- Use doubled [r]esolution:
+
+`catimg -r 2 {{path/to/file}}`
```

Comment:
```suggestion
`catimg -r {{2}} {{path/to/file}}`
```

Reply:
"2" is the only value that matters. The only other option is "1", which is the default. So I'm basically just showing the only possible use of the `-r` flag.